### DISCUSSION TITLE
About the certificate in Fluentbit/Fluentd

### DISCUSSION THREAD

**Discussion by i5Js** [ricsanfre/pi-cluster#364] [Q&A] [answered]
Hey Ricardo, 

I have a certificate created for my home network: *.home.domain.com. 

Should I need to create a new one only for fluentd.home.domain.com?

Thanks in advanced,

**Comment by ricsanfre** [comment, marked as answer]
Hola Javi,

I do not use wildcard certificates in my cluster, I use Cert-Manager to create certificates for every exposed endpoint. I suppose wildcard certificates should work but I never tested it. Let me know if it works.

**Reply by i5Js** [reply]
It should work, but I won't know it till I can fix the issue between fluent and elastic
